spec/mailers/user_notifications_spec.rb

```diff
@@ -78,7 +78,33 @@
       expect(subject.from).to eq([SiteSetting.notification_email])
       expect(subject.body).to be_present
     end
+  end
+
+  describe ".confirm_new_email" do
+    let(:opts) { { requested_by_admin: requested_by_admin, email_token: token } }
```

Comment:
Double braces can be confusing in this case, because one represents a block and one represents a hash. I suggest making this a multi line `do ... end` block to make it more clear.